access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp

```diff
@@ -35,19 +35,24 @@
 
 <span class="hierarchicalTrail">  
 	<c:forEach items="${objectPath.entries}" var="pathEntry" varStatus="status">
-		<c:if test="${!param.hideLast || !status.last}">
 			<c:if test="${!status.first}">
 				&gt; 
 			</c:if>
 			<c:choose>
-				<c:when test="${status.last && param.linkLast != true}">
+				<c:when test="${status.last}">
 					<c:out value="${pathEntry.name}" />
 				</c:when>
 				<c:otherwise>
-					<c:url var="shiftFacetUrl" scope="page" value="${queryPath}/${pathEntry.pid}${shiftFacetUrlBase}"></c:url>
+					<c:choose>
+						<c:when test="${pathEntry.resourceType == searchSettings.resourceTypeAggregate}">
+							<c:url var="shiftFacetUrl" scope="page" value="record/${pathEntry.pid}${shiftFacetUrlBase}"></c:url>
```

Comment:
Its not doing any harm, but you can safely remove `${shiftFacetUrlBase}`.  That variable adds facet and keyword parameters from the users current search, which wouldn't do anything on a full record page